draft-ietf-cose-cwt-claims-in-headers.md

```diff
@@ -69,6 +69,8 @@ CWT-Claims = {
 Claim-Label = int / text
 ```
 
+It is RECOMMENDED that the CWT claims header parameter is used only in a protected header to avoid the contents from being malleable. The header parameter MUST only occur once in either the protected or unprotected header of a COSE structure.
```

Comment:
```suggestion
It is RECOMMENDED that the CWT claims header parameter is used only in a protected header to avoid the contents being malleable. The header parameter MUST only occur once in either the protected or unprotected header of a COSE structure.
```